deployment/terraform/cdn.tf

```diff
@@ -0,0 +1,88 @@
+module "origin" {
+  source               = "github.com/azavea/terraform-aws-s3-origin?ref=0.2.0"
+  bucket_name          = "tilejson-io-${lower(var.environment)}-site-${var.aws_region}"
+  logs_bucket_name     = "tilejson-io-${lower(var.environment)}-logs-${var.aws_region}"
+  cors_allowed_headers = ["Authorization"]
+  cors_allowed_methods = ["GET"]
+  cors_allowed_origins = ["*"]
+  cors_max_age_seconds = "3000"
+  region               = "${var.aws_region}"
+
+  project     = "${var.project}"
+  environment = "${var.environment}"
+}
+
+resource "aws_cloudfront_distribution" "app" {
+  origin {
+    domain_name = "${module.origin.site_bucket}.s3.amazonaws.com"
+    origin_id   = "originEastId"
+    origin_path = ""
+  }
+
+  enabled             = true
+  http_version        = "http2"
+  comment             = "TILEJSON IO (${var.environment})"
+  default_root_object = "index.html"
+  retain_on_delete    = true
+
+  price_class = "PriceClass_All"
+  aliases     = ["${var.r53_public_hosted_zone}"]
+
+  default_cache_behavior {
+    allowed_methods  = ["GET", "HEAD", "OPTIONS"]
+    cached_methods   = ["GET", "HEAD", "OPTIONS"]
+    target_origin_id = "originEastId"
+
+    forwarded_values {
+      query_string = false
+
+      cookies {
+        forward = "none"
+      }
+    }
+
+    compress               = true
+    viewer_protocol_policy = "allow-all"
```

Comment:
Now that we have a valid certificate, we can change this value to `redirect-to-https`.